ui/app/img/caret-right.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="8" height="14" viewBox="0 0 8 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
That's not really needed. You can use the normal caret and rotate it by 90 degrees.
If you need to change the color, you just need a `ColorOverlay`